### PR TITLE
new post: add 2023 sponsor cathay post

### PR DESCRIPTION
**Pull request by kevin1kevin1k**
## Types of changes

* new post

## Description

Add a new post for platinum sponsor Cathay of PyCon TW 2023
Original draft at [google docs](https://docs.google.com/document/d/1S4XpIcCZr5c0Zta9ld97nZcsM-SU-_aeJ0viq-qOjxI/edit?usp=sharing)

### CODE REVIEW

File: content/posts/2023-10-25-pycon-tw-2023-sponsor-cathay.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## 公司介紹
+
+國泰金控以成為亞洲最佳金融機構為目標，除了在國內市場穩居領導地位，對外亦持續積極地朝亞太地區拓點，藉由打造區域平台的核心策略以提供客戶整合性的金融服務。同時也致力於推動集團數位轉型，以數據驅動、突破創新為理念，協助集團各子公司打造跨界金融，一步到位的金融服務。國泰金控數位數據暨科技發展中心長期投入大量資源，專注於資料科學領域研發，與旗下各子公司探討資料科學技術的創新應用，以數據驅動金融科技創新，提供有感、有效的客戶數位體驗，致力成為亞洲數位生態圈及科技金融領頭羊。
+
+
+## 精選職缺
+
+| 公司                | 職缺                                           | 連結                                                          |   |   |
+|---------------------|------------------------------------------------|---------------------------------------------------------------|---|---|
+| 國泰金控DDT         | 資料科學工程師(Python程式開發)                 | <a href="https://www.104.com.tw/job/6w7ov" target="_blank">https://www.104.com.tw/job/6w7ov</a> |   |   |
```
Comment (Lee-W) [inline content/posts/2023-10-25-pycon-tw-2023-sponsor-cathay.md:22]: May I know why we don't use `[]()` syntax for links here?

**Review by winniepopu — APPROVED** [review]
LGTM, Thank you!

File: content/posts/2023-11-12-pycon-tw-2023-sponsor-cathay.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+Title: PyCon Taiwan 2023 贊助商 國泰金控 Cathay Financial Holdings
+Date: 2023-11-12 21:00
```
Comment (Lee-W) [inline content/posts/2023-11-12-pycon-tw-2023-sponsor-cathay.md:2]:
```suggestion
Date: 2023-11-20 21:00
```

File: content/posts/2023-11-12-pycon-tw-2023-sponsor-cathay.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+|       國泰產險      | 商業數據分析師                                 | [連結](https://www.104.com.tw/job/6l49y)                              |   |   |
+| 台中資訊開發中心CDC | 資料科學前端工程師 (Frontend Engineer)         | [連結](https://www.104.com.tw/job/80xzh)                              |   |   |
+| 台中資訊開發中心CDC | 資深資料科學前端工程師 (Sr. Frontend Engineer) | [連結](https://www.104.com.tw/job/80xz7)                              |   |   |
+| 台中資訊開發中心CDC | 數據工程師                                     | [連結](https://www.104.com.tw/job/763vo)                              |   |   |
+| 台中資訊開發中心CDC | 資料科學家                                     | [連結](https://www.104.com.tw/job/763vy)                              |   |   |
+| 高雄資訊開發中心CDC | 雲端數據工程師                                 | [連結](https://www.104.com.tw/job/7sf4c)        |   |   |
+| 高雄資訊開發中心CDC | 專案管理師                                     | [連結](https://www.104.com.tw/job/81b74)        |   |   |
+|                     |                                                |                                                               |   |   |
```
Comment (Lee-W) [inline content/posts/2023-11-12-pycon-tw-2023-sponsor-cathay.md:44]:
```suggestion
```

**Review by Lee-W — CHANGES_REQUESTED** [review]

**Review by Lee-W — APPROVED** [review]
I just resolved my comments on my side. @kevin1kevin1k Thanks for helping out.

**Pull request merged**